packages/mui-base/src/ClickAwayListener/ClickAwayListener.test.js

```diff
@@ -25,6 +25,7 @@ describe('<ClickAwayListener />', () => {
    * React bug: https://github.com/facebook/react/issues/20074
    */
   function render(...args) {
+    // eslint-disable-next-line testing-library/render-result-naming-convention
```

Comment:
The [`render-result-naming-convention`](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/render-result-naming-convention.md) rule seems to warn whenever it finds a function/method name that contains "render" 😅

Reply:
oh 🤦 .